Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/AuthMethodManager.java

```diff
@@ -83,7 +93,7 @@ private AuthMethodManager() {
             try {
                 initAuthMethodManagerFromSettings();
             } catch (Throwable ex) {
-                LOGGER.warning("Cannot restore login due to error: " + ex.getMessage());
+                LOGGER.warn("Cannot restore login due to error: " + ex.getMessage());
```

Comment:
use AzureMessager if this message is to show to user.

Reply:
will refactor a lot of such cases in next sprint